CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 Mapbox welcomes participation and contributions from everyone.
 
 # main
+## Bug fixes 🐞
+* Fix a bug where `ViewAnnotationManager.annotations` API doesn't return correct camera bounds. ([1861](https://github.com/mapbox/mapbox-maps-android/pull/1861))
```

Comment:
yes. I can add that to v10.10 changelog when creating port pr .

Reply:
@ank27 issue with your approach is that in that case changlelog will be correct in `v10.10` branch but it will be incorrect in `main` branch and this record might be duplicated in `v10.11-beta.1` then. 
I would advise to rebase now and add it to the correct section.

Reply:
sure, will do :)